test/parallel/test-release-changelog.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+// This test checks that the changelogs contain an entry for releases.
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const getDefine = (text, name) => {
+  const regexp = new RegExp(`#define\\s+${name}\\s+(.*)`);
+  const match = regexp.exec(text);
+  assert.notStrictEqual(match, null);
+  return match[1];
+};
+
+const srcRoot = path.join(__dirname, '..', '..');
+const mainChangelogFile = path.join(srcRoot, 'CHANGELOG.md');
+const versionFile = path.join(srcRoot, 'src', 'node_version.h');
+const versionText = fs.readFileSync(versionFile, { encoding: 'utf8' });
+const release = getDefine(versionText, 'NODE_VERSION_IS_RELEASE') !== '0';
+
+if (!release) {
+  common.skip('release bit is not set');
+}
+
+const major = getDefine(versionText, 'NODE_MAJOR_VERSION');
+const minor = getDefine(versionText, 'NODE_MINOR_VERSION');
+const patch = getDefine(versionText, 'NODE_PATCH_VERSION');
+const version = `${major}.${minor}.${patch}`;
```

Comment:
```suggestion
const version = `${major}.${minor}.${patch}`;
const versionForRegex = `${major}\\.${minor}\\.${patch}`;
```

Reply:
Pushed a fixup for this. `version` is no longer used afterwards (as it was only used in the regular expressions) so I removed it.